<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRecipe</class>
 <widget class="QDialog" name="AddRecipe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(30, 30, 30);
selection-background-color: rgb(104, 104, 104);

background-color: rgb(62, 62, 62);</string>
  </property>
  <widget class="QDialogButtonBox" name="AcceptRecipe">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>121</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="VeganBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(36, 36, 36);</string>
      </property>
      <property name="text">
       <string>Vegan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="VegitarianBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(36, 36, 36);</string>
      </property>
      <property name="text">
       <string>Vegitarian</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="LactoseBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(36, 36, 36);</string>
      </property>
      <property name="text">
       <string>Lactose Intolerant</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="FishBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(36, 36, 36);</string>
      </property>
      <property name="text">
       <string>Fish</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="GlutenBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(36, 36, 36);</string>
      </property>
      <property name="text">
       <string>Gluten</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>16</width>
     <height>171</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Calories</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CaloriesLineEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 36, 36);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="IngredientList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>251</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 36, 36);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="NameInsert">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(36, 36, 36);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="IngredientInsert">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(36, 36, 36);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Ingredient Insert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddIngredient">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Recipe Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Recipe Name:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AcceptRecipe</sender>
   <signal>accepted()</signal>
   <receiver>AddRecipe</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AcceptRecipe</sender>
   <signal>rejected()</signal>
   <receiver>AddRecipe</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
